04/app.js

```diff
@@ -1,5 +1,43 @@
 document.addEventListener('DOMContentLoaded', init);
 
 function init() {
-    console.log('DOM');
-}
\ No newline at end of file
+	console.log('DOM');
+	const formEl = document.querySelector('form');
+	formEl.addEventListener('submit', getWeather);
+}
+
+function getWeather(e) {
+	e.preventDefault();
+
+	const latEl = document.querySelector('.form__field--lat');
+	const lat = latEl.value;
+	const lngEl = document.querySelector('.form__field--lng');
+	const lng = lngEl.value;
+	const latData = document.querySelector('.weather__lat');
+	const lngData = document.querySelector('.weather__lng');
+	latData.innerText = lat;
+	lngData.innerText = lng;
+	const weatherEl = document.querySelector('.weather__summary');
+	const tempEl = document.querySelector('.weather__temperature');
```

Comment:
Dużo się tutaj dzieje. Lepiej byłoby podzielić to na kilka funkcji.

1) Taka która pobiera dane z formularza
2) Kolejna funkcja pobiera dane z API, a współrzędne pobiera przez parametr
3) Taka która ustawia wartości w odpowiednich elementach DOM

Wtedy będzie można taki kod wykorzystać w wielu miejscach :)